src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/FootnoteServiceTests.cs

```diff
@@ -1205,13 +2099,135 @@ await using (var statisticsDbContext = InMemoryStatisticsDbContext(contextId))
             }
         }
 
+        private static async Task SeedDatabase(
+            string contextId,
+            Release release,
+            IReadOnlyList<Subject>? subjects = null,
+            IReadOnlyList<ReleaseSubject>? releaseSubjects = null,
+            IReadOnlyList<Footnote>? footnotes = null)
+        {
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(contextId))
+            {
+                await statisticsDbContext.Release.AddAsync(release);
+
+                if (!subjects.IsNullOrEmpty())
+                {
+                    await statisticsDbContext.Subject.AddRangeAsync(subjects!);
+                }
+
+                if (!releaseSubjects.IsNullOrEmpty())
+                {
+                    await statisticsDbContext.ReleaseSubject.AddRangeAsync(releaseSubjects!);
+                }
+
+                if (!footnotes.IsNullOrEmpty())
+                {
+                    await statisticsDbContext.Footnote.AddRangeAsync(footnotes!);
+                }
+
+                await statisticsDbContext.SaveChangesAsync();
+
+                await contentDbContext.AddAsync(new Content.Model.Release
+                {
+                    Id = release.Id
+                });
+                await contentDbContext.SaveChangesAsync();
+            }
+        }
+
+        private static async Task<Either<ActionResult, Footnote>> CreateFootnoteWithConfiguration(
+            Guid releaseId,
+            string contextId,
+            string? content = null,
+            IReadOnlySet<Guid>? filterIds = null,
+            IReadOnlySet<Guid>? filterGroupIds = null,
+            IReadOnlySet<Guid>? filterItemIds = null,
+            IReadOnlySet<Guid>? indicatorIds = null,
+            IReadOnlySet<Guid>? subjectIds = null)
+        {
+            content ??= string.Empty;
+            filterIds ??= SetOf<Guid>();
+            filterGroupIds ??= SetOf<Guid>();
+            filterItemIds ??= SetOf<Guid>();
+            indicatorIds ??= SetOf<Guid>();
+            subjectIds ??= SetOf<Guid>();
```

Comment:
I did this mainly so that I didn't have to specify the input parameters everywhere if they were intended to be empty lists. I only pass them in if they are populated. Just keeps the method calls a bit tidier

Reply:
Chatted through this with Jack to explain with more context the change I was intending, and we decided not to make any change here.